contents/docs/data/anonymous-vs-identified-events.mdx

```diff
@@ -148,6 +146,50 @@ Calling [`reset()`](/docs/product-analytics/identify#3-reset-after-logout) will
+
+Yes, identified events can still be "anonymous" in the sense that they don't need to contain personal information such as name, email, or phone number. A person profile is still created for them, but it doesn't need to include any personal information.
+
+### What happens to initial person properties when an anonymous user is identified? Do we get the initial person properties from the anonymous events?
+
+As person properties are not stored on anonymous events, we cannot get any initial person properties directly from those events. We store initial URL and initial referrer in [your persistence](docs/libraries/js#persistence) and from this we are able to derive the following values which are set when the person profile is created:
+ - `$initial_current_url`
+ - `$initial_pathname`
+ - `$initial_referrer`
+ - `$initial_referring_domain`
+ - `$initial_host`
+ - and any initial properties that can be derived from the URL i.e 
+    - `$initial_utm_campaign`
+    - `$initial_utm_content`
+    - `$initial_utm_medium`
+    - `$initial_utm_source`
+
+<blockquote>
+Note that the above initial parameters will only work across subdomains if you're using `persistence: "localStorage+cookie"` (default) or `persistence: "cookie"`
+</blockquote>
+
+Any other initial [person properties](product-analytics/person-properties#default-person-properties), including initial [GeoIP properties](https://posthog.com/docs/product-analytics/person-properties#geoip-properties) will be set from the values in the event at the time that the person profile was created (i.e. when the events became identified).
+
+### If we change our `person_profiles` config to `'always'` away from `'identify-only'`? Will we get all the person data from the previously captured anonymous events?
+
+Anonymous events only capture the event data and none of the person data. If that person becomes identified at a later date, either by identifying them or changing your configuration settings, there isn't a way to retrieve what the person properties would have been at the time of those anonymous events.
+
+<!---  Insights questions -->
+
+### Why am I seeing "Person without distinct_id" in my insights?
+
+These are users who do not have a person profile and are anonymous users. We're aware that the wording here is confusing and are looking into updating this.
+
+### Why am I seeing drops in my insights that rely on person properties?
+
+If your insight is using a person property, then this relies on identified events. When using anonymous events, the person profile will not exist until the person has been identified and the events have become identified events. In some cases you may never identify a person, and in some cases you may identify a person at a later date.
+
+Depending on what you're trying to visualize, you may be able to use some of the properties on the events to achieve a similar outcome i.e. Looking at the referring domain on the first ever pageview for each user.
```

Comment:
I dont really understand this answer. It doesnt really answer why there is a "drop" ?

Reply:
Basically if they have previously been using the default of `always` and they have now been switched to the default of `identified_only`, we are seeing some questions about 'drops' in the trends insight for example. If they have a trend insight looking at events which is filtering on a person property, this will see a drop at the time that we switched over the default to `identified_only` as those events no longer have person properties by default. Does that make sense? Maybe it's not super useful as a question at the moment as it should just be an immediate issue at the time of the switch over?

Reply:
I've tried to update the wording here. If you still think it's unclear then I may remove this question from the FAQ page altogether.